README.md

```diff
@@ -483,6 +484,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayush Raj](https://github.com/imyoungsparda)
   - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
   - [Aziz Prabowo](https://github.com/azizp128)
+  
```

Comment:
Hey @AnthonyByansi ,
Thanks for your contribution :tada: . You put an extra new line here. Please remove this.